monkey/agent_plugins/exploiters/powershell/src/powershell_remote_access_client.py

```diff
@@ -81,7 +81,7 @@ def execute_agent(self, agent_binary_path: PurePath, tags: Set[AgentEventTag]):
         try:
             tags.update(EXECUTION_TAGS)
             self._powershell_client.execute_cmd_as_detached_process(
-                self._command_builder(self.get_os(), agent_binary_path)
+                self._command_builder(agent_binary_path)
```

Comment:
Have you tested? I think you will need to define the argument's name: `remote_agent_binary_destitination_path`.

Reply:
Nope. Why? It worked without it previously.

Reply:
It is a single argument. I know it didn't work when we did SSH.

Reply:
EtE passes.